cmd/talosctl/pkg/talos/action/tracker.go

```diff
@@ -189,6 +191,38 @@ func (a *Tracker) Run() error {
 		Backoff:           backoff.Config{},
 		MinConnectTimeout: 20 * time.Second,
 	}))
+
+	err = multierror.Append(err, eg.Wait())
+
+	if !a.debug {
+		return err
+	}
+
+	var failedNodes []string
+
+	failedNodesToDmesgs.Range(func(key, value any) bool {
```

Comment:
Maybe we can replace `sync.Map` with `github.com/siderolabs/gen/containers.ConcurrentMap`?

Reply:
nice 👍

Reply:
done